ANMN/temperature_grid_product/README.md

```diff
@@ -0,0 +1,28 @@
+# anmn_temp_gridded_product.py
+
+This piece of Python 2.7 code creates a gridded temperature products based on FV01 NetCDF files for a particular deployment_code:
```

Comment:
Worth adding that FV01 means quality-controlled, and also that these are single-instrument files.